net-ftp/pureadmin/files/pureadmin-0.4-gold.patch

```diff
@@ -13,8 +7,6 @@ index 94f61e0..0ebb561 100644
 -pureadmin_LDADD = @GTK_LIBS@ $(INTLLIBS) @BINRELOC_LIBS@
 +pureadmin_LDADD = @GTK_LIBS@ $(INTLLIBS) @BINRELOC_LIBS@ -lm -lX11
```

Comment:
no `{R,}DEPEND` on libX11?

Reply:
Oh, good catch, I guess that was forgotten by whoever added the patch. Done, and also revbumped